verbcalc/core/calculate.py

```diff
@@ -19,16 +20,22 @@ def calculate(sentence: str,
 
         dispatcher:
             Dispatcher object to use, if none provided it will use default one.
+        
+        silent:
+            Returns only the answer instead of an entire answer sentence
 
     Returns:
         Calculated sentence.
     """
 
     try:
         result = dispatcher.dispatch(translate(sentence).split())
-        if result % 1 == 0:
-            return 'The result is ' + str(int(result))
-        else:
-            return 'The result is ' + str(result)
+        
+        answer = str(int(result)) if result % 1 == 0 else str(result)
+        
+        return answer if silent else ' '.join([CustomAnswers().get_phrase(default=True), answer])
```

Comment:
I know it made testing easier, but let's not have `default=True` as library default.

```suggestion
        return answer if silent else ' '.join([CustomAnswers().get_phrase(), answer])
```